ecosystem/sep-0024.md

```diff
@@ -344,13 +413,13 @@ When a successful response comes back (either from polling or via callback), the
 The wallet must use the response fields in the following way to complete the withdrawal:
 
 - `status`: `pending_user_transfer_start` means the user has given all necessary info to the anchor, and it is up to the wallet to send the actual stellar assets.
-- `withdraw_anchor_account`: send the withdrawal payment to this Stellar account. 
+- `withdraw_anchor_account`: send the withdrawal payment to this Stellar account.
```

Comment:
Seems like there's a view formatting issues in this PR that could be fixed independent of a major change to the SEP. Could they be moved to a separate PR we can merge quickly?